dask/array/utils.py

```diff
@@ -583,17 +530,3 @@ def scipy_linalg_safe(func_name, *args, **kwargs):
 
 def solve_triangular_safe(a, b, lower=False):
     return scipy_linalg_safe("solve_triangular", a, b, lower=lower)
-
-
-def _is_nep18_active():
-    class A:
-        def __array_function__(self, *args, **kwargs):
-            return True
-
-    try:
-        return np.concatenate([A()])
-    except ValueError:
-        return False
-
-
-IS_NEP18_ACTIVE = _is_nep18_active()
```

Comment:
Suppose that's possible. Figured since it was default "on" and has been included for a while now (since NumPy 1.17) that the risk here was low. The [note in NEP 18]( https://numpy.org/neps/nep-0018-array-function-protocol.html#implementation ) suggests the environment variable may leave in the future, "eventually, expect to `__array_function__` to always be enabled."

Reply:
@pentschev wdyt?

Reply:
Agreed it's low risk and probably not worth the maintenance burden of supporting `NUMPY_EXPERIMENTAL_ARRAY_FUNCTION`. I'm happy to keep this PR as it currently is and move on 👍

Reply:
I think it's very low risk as well, so I'm up for dropping that flag as well.

Reply:
Ok cool. Thanks all! 😄